main.tf

```
@@ -184,6 +184,11 @@ module "redis" {
   kms_key_arn = local.kms_key_arn
 }
 
+locals {
+  max_lb_name_length = 32 - length("-alb-k8s")
+  lb_name_truncated  = "${substr(var.namespace, 0, min(length(var.namespace), local.max_lb_name_length))}-alb-k8s"
```

```suggestion
  lb_name_truncated  = "${substr(var.namespace, 0, local.max_lb_name_length)}-alb-k8s"
```
I don't think we need the min